library/std/src/sys/unix/fs.rs

```diff
@@ -1020,8 +1074,8 @@ impl OpenOptions {
 }
 
 impl File {
-    pub fn open(path: &Path, opts: &OpenOptions) -> io::Result<File> {
-        run_path_with_cstr(path, |path| File::open_c(path, opts))
+    pub fn open_native(path: &CStr, opts: &OpenOptions) -> io::Result<File> {
```

Comment:
This can probably be removed now, as it's identical to `open_c`.

Reply:
I don't think it would add much size, it's just a rename from `open_c` to `open_native`. For me, at least, that's actually easier to read.

Reply:
Huh, actually there's only one use of `open_c` I can find. So you're right, that's not an issue.